src/components/activity/ActivityTableMenu.svelte

```diff
@@ -0,0 +1,163 @@
+<svelte:options immutable={true} />
+
+<script lang="ts">
+  import SearchIcon from '@nasa-jpl/stellar/icons/search.svg?component';
+  import type { ColDef, ColumnState } from 'ag-grid-community';
+  import { createEventDispatcher } from 'svelte';
+  import Input from '../form/Input.svelte';
+  import Menu from '../menus/Menu.svelte';
+  import MenuItem from '../menus/MenuItem.svelte';
+
+  export let columnDefs: ColDef[] = [];
+  export let columnStates: ColumnState[] = [];
+
+  interface ColumnMenuItem {
```

Comment:
Can we move this to types since it's used more than once? Also it should probably be `type` since you are not extending it.

Reply:
Ah the other `ColumnMenuType` was accidentally left in because I moved it over to its own file. I'll remove the duplicate, but do you think it still needs to go into types?

Reply:
Ah nice. It's fine I think for now.